.github/workflows/update-lib.yml

```
@@ -0,0 +1,66 @@
+name: "Update lib/"
+
+on:
+  workflow_dispatch:
+    # Example of how to trigger this using the API:
+    # curl -X POST \
+    #      -H "Authorization: Bearer <token>" \
+    #      https://api.github.com/repos/github/codeql-action/actions/workflows/update-lib.yml/dispatches \
+    #      -d '{"ref":"main","inputs":{"branch":"<branch name>"}}'
+    # Replace <token> with a personal access token from this page: https://github.com/settings/tokens
+    # Replace <pr-number> by the number of the pull request to update
```

```suggestion
    # Replace <branch name> by the name of the branch of the PR.
```

Did you test this for forks? Should it then be `<user>:<branch>`?

---

I did not test this for forks and you're right this isn't going to work as it is.